model/annotations.go

```diff
@@ -343,6 +343,12 @@ type PdfAnnotationWidget struct {
 	processing bool // Used in ToPdfObject serialization to avoid infinite loops for merged-in annots.
 }
 
+// Field returns the parent form field of the widget annotation, if one exists.
+// NOTE: the method returns nil if the parent form field has not been parsed.
```

Comment:
Would it make sense to parse it if it has not been parsed?  or output a debug message if we can tell that it has not been parsed ?  In what case can this happen?

Reply:
This can happen if the AcroForm is broken (the fields are not present in the AcroForm). In this case, only the annotations are parsed by the reader, when building the page list. This can also happen if a user creates a widget annotation manually through the `NewPdfAnnotationWidget` function. So it's a bit tricky to determine the cause of the uninitialized parent field.

The fields are parsed when parsing the AcroForm, in the `newPdfAcroFormFromDict` method of the reader. There's a call to `reader.newPdfFieldFromIndirectObject` for each element in the `Fields` field of the AcroForm container.

There's no reference to the reader in the `PdfAnnotationWidget` struct, so there's no way of calling `newPdfFieldFromIndirectObject` from the `Field` method.